specs/eip4844/p2p-interface.md

```diff
@@ -65,34 +78,35 @@ The new topics along with the type of the `data` field of a gossipsub message ar
 
 | Name | Message Type |
 | - | - |
-| `beacon_block_and_blobs_sidecar` | `SignedBeaconBlockAndBlobsSidecar` (new) |
+| `blob_sidecar_{index}` | `SignedBlobSidecar` (new) |
 
 #### Global topics
 
-EIP-4844 introduces a new global topic for beacon block and blobs-sidecars.
+EIP-4844 introduces new global topics for blob sidecars.
 
 ##### `beacon_block`
 
-This topic is deprecated and clients **MUST NOT** expose in their topic set to any peer. Implementers do not need to do
-anything beyond simply skip implementation, and it is explicitly called out as it is a departure from previous versioning
-of this topic.
+The *type* of the payload of this topic changes to the (modified) `SignedBeaconBlock` found in EIP4844.
 
-Refer to [the section below](#transitioning-the-gossip) for details on how to transition the gossip.
+##### `blob_sidecar_{index}`
 
-##### `beacon_block_and_blobs_sidecar`
+This topic is used to propagate signed blob sidecars, one for each sidecar index.
 
-This topic is used to propagate new signed and coupled beacon blocks and blobs sidecars to all nodes on the networks.
+The following validations MUST pass before forwarding the `sidecar` on the network, assuming the alias `sidecar = signed_blob_sidecar.message`:
 
-In addition to the gossip validations for the `beacon_block` topic from prior specifications, the following validations MUST pass before forwarding the `signed_beacon_block_and_blobs_sidecar` on the network.
-Alias `signed_beacon_block = signed_beacon_block_and_blobs_sidecar.beacon_block`, `block = signed_beacon_block.message`, `execution_payload = block.body.execution_payload`.
-- _[REJECT]_ The KZG commitments correspond to the versioned hashes in the transactions list
-  -- i.e. `verify_kzg_commitments_against_transactions(block.body.execution_payload.transactions, block.body.blob_kzg_commitments)`
-
-Alias `sidecar = signed_beacon_block_and_blobs_sidecar.blobs_sidecar`.
-- _[IGNORE]_ the `sidecar.beacon_block_slot` is for the current slot (with a `MAXIMUM_GOSSIP_CLOCK_DISPARITY` allowance)
-  -- i.e. `sidecar.beacon_block_slot == block.slot`.
-- _[REJECT]_ The KZG commitments in the block are valid against the provided blobs sidecar
-  -- i.e. `validate_blobs_sidecar(block.slot, hash_tree_root(block), block.body.blob_kzg_commitments, sidecar)`
+- _[REJECT]_ The sidecar is for the correct topic --
+  i.e. `sidecar.index` matches the topic `{index}`.
+- _[IGNORE]_ The sidecar is not from a future slot (with a `MAXIMUM_GOSSIP_CLOCK_DISPARITY` allowance)
+- _[IGNORE]_ The sidecar is from a slot greater than the latest finalized slot --
+  i.e. validate that `sidecar.slot > compute_start_slot_at_epoch(state.finalized_checkpoint.epoch)`
+- _[REJECT]_ The proposer signature, `signed_blob_sidecar.signature`, is valid with respect to the `sidecar.proposer_index` pubkey.
+- _[IGNORE]_ The sidecar is the only sidecar with valid signature received for the tuple `(sidecar.slot, sidecar.proposer_index, sidecar.index)`.
```

Comment:
I would consider upgrade this to `REJECT` if we are implying that the block with multiple sidecars *MUST* be discarded. If we say `*MAY*` discard then `IGNORE` makes sense

Reply:
Ah, but it is not attributable that the sender knew that there was a duplicate (or that you already got it) before sending. Thus it has to be IGNORE as it can't strictly be downscored